packages/vitest/src/node/stdin.ts

```diff
@@ -119,6 +119,11 @@ export function registerConsoleShortcuts(
       return inputFilePattern()
     }
     if (name === 'b') {
+      await Promise.all(ctx.projects.map(async (project) => {
+        if (project.browser) {
```

Comment:
This will just close the browser, it won't be recreated since init checks for existence of browser, not on its state

Reply:
```ts
// FIXME: should we close the orchestrators and testers?
// project.browser.state.orchestrators.forEach(orchestrator => orchestrator.$close())
// project.browser.state.orchestrators.clear()
// project.browser.state.testers.forEach(tester => tester.$close())
// project.browser.state.testers.clear()
```

 Should we also clear the `orchestrators and testers`? Right now, clearing the orchestrators poses no issues, but because we don't have a type declaration for testers, adding support for clearing the would require a large refactor.
I look forward to team's feedback.